_studio/mfx_lib/scheduler/include/mfx_scheduler_core.h

```diff
@@ -312,6 +313,31 @@ class mfxSchedulerCore : public MFXIScheduler2
     inline MFX_SCHEDULER_THREAD_CONTEXT* GetThreadCtx(mfxU32 thread_id)
     { return &m_pThreadCtx[thread_id]; }
 
+    // Invokes functor 'bool F(MFX_SCHEDULER_TASK*)' for every valid task that returns 'false' to continue iteration or 'true' to stop it.
+    template <typename F>
+    void ForEachTask(F&& f)
+    {
+        mfxU32 priority;
```

Comment:
minor: can move all of these definitions in place to make code more compact

Reply:
ok